core/packages/test/.envrc-example

```diff
@@ -1,4 +1,4 @@
-source_up_if_exists
+# source_up_if_exists
```

Comment:
What is this for? Since it is commented out, perhaps we can remove it?

Reply:
I do not know what it is for and guess it is just some hint to source the env. Since we've introduced `direnv` and that will be done automaticlly so would assume we do not need it.
So if no other comment here I will just remove it.

Reply:
This runs the root `.envrc`, if it exists. It's for the [nix-direnv integration](https://github.com/Snowfork/snowbridge/blob/main/.envrc.example). The end of the [Development section of the root README](https://github.com/Snowfork/snowbridge/blob/main/README.md#development) shows how to set that up.

When the direnv integration is set up in the root and there's an `.envrc` in the current directory without this line, we'll leave the Nix shell. This removes our dev dependencies, which definitely isn't what we want in the test directory.

[direnv docs](https://direnv.net/man/direnv-stdlib.1.html#codesourceupifexists-ltfilenamegtcode)

Reply:
Nice.